x-pack/plugin/data-frame/src/test/java/org/elasticsearch/xpack/dataframe/persistence/DataFrameTransformsConfigManagerTests.java

```diff
@@ -239,7 +239,7 @@ public void testStateAndStats() throws InterruptedException {
     }
 
     public void testGetStateAndStatsMultiple() throws InterruptedException {
-        int numStats = randomInt(5);
+        int numStats = randomIntBetween(10, 15);
```

Comment:
Is this change related to the one above? If so, how?

Reply:
The default number of search results is 10. The bug is that the default value was used so only the first 10 documents are returned no matter how many stats documents there are. If this test was using a number > 10 in the first place the mistake would have been caught.

Reply:
Could you add a line comment here? Something like:
// Allow numStats greater than 10 so that...